android/src/main/java/com/auth0/react/A0Auth0Module.kt

```
@@ -293,34 +295,41 @@ class A0Auth0Module(private val reactContext: ReactApplicationContext) : A0Auth0
                         authAPI,
                         reactContext,
                         auth0!!,
-                        SharedPreferencesStorage(reactContext),
+                        storage,
                         activity,
                         localAuthOptions
                     )
                     promise.resolve(true)
                     return
                 } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)